app/views/coronavirus_pages/show.html.erb

```diff
@@ -21,7 +21,9 @@
 
 <div class="covid-manage-page govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% if unreleased_feature_user? && @coronavirus_page.topic_page? %>
```

Comment:
Sorry, I missed this in an earlier review, but I think there should be a test for the `topic_page?` guard, similar to the one for [announcements](https://github.com/alphagov/collections-publisher/blob/master/spec/features/coronavirus_page_spec.rb#L222) to make sure that the timeline section isn't being displayed on hub pages as it would be very easy to accidentally delete the guard when the timeline feature is made live.